index.html

```diff
@@ -25,27 +25,27 @@ <h3>I’m a software developer! </h3>
             <ul class="social_icons flex">
                 <li>
                     <a href="#">
-                        <img src="/images/Icon -GitHub.png" alt="github logo"/>
+                        <img src="/images/Icon-GitHub.png" alt="github logo"/>
                     </a>
                     </li>
                 <li>
                     <a href="#">
-                        <img src="/images/Icon - Linkedin.png" alt="linkedin logo" />
+                        <img src="/images/Icon-Linkedin.png" alt="linkedin logo" />
                     </a>
                 </li>
                     <li>
                         <a href="#">
-                            <img src="/images/Icon - Angelist.png" alt="angelist logo"/>
+                            <img src="/images/Icon-Angelist.png" alt="angelist logo"/>
```

Comment:
- [ ] As suggested by the previous reviewer. I would kindly suggest you work on all the social media links as currently the links are broken. I noticed all the images are not in your project folder 👍 👍 

Your project 👇 

![dara 1](https://user-images.githubusercontent.com/78634841/212937058-a4479b1e-da05-4965-8230-0b7a33bea9c0.png)